### ISSUE TITLE
Feature: Use real "Real time" architecture for real time map

### ISSUE THREAD

**Issue by saerdnaer** [WRSC/tracking#35]
For the next installation of this event series I would recommend to switch the realtime views to a message queue/browser e.g. MQTT and connect the web clients via Websocket.

When designing the MQTT topics, one could also reuse existing schemes to be able to use Owntracks on iOS/Andorid to follow the boats, c.f. https://owntracks.org/booklet/guide/broker/

**Comment by takluyver** [comment]
I would sound a note of caution about this, on two counts. Firstly, it sounds like a Big Rewrite, and there's always a risk of introducing new problems with those. Secondly, I think the tracking system is not always run by people who're familiar with running websites and network services, so introducing an MQTT broker might make it significantly harder to set up.

I may be overreacting - I tend to get conservative about software projects. I can definitely see that the realtime mapping interface could be much better, and I don't want to stop people improving it. But we should think about whether a rewrite would actually be completed, and what ongoing support it might require.

**Comment by saerdnaer** [comment]
Yes, it should be an optional addon. But as we now live in the world of containers and kubernetes deploying software is not such an big issue as before. But this is actually a seperated discussion -> #46
